Cargo.toml

```diff
@@ -102,7 +106,7 @@ rayon = "1.8.0"
 rlimit = "0.10.2"
 rustc_version_runtime = { version = "0.3.0", default-features = false }
 reqwest = { version = "0.12.5", features = ["rustls-tls", "json", "http2"], default-features = false }
-rocksdb = { version = "0.22.0", features = ["lz4"] }
+rocksdb = { version = "0.22.0", features = ["lz4"], default-features = false }
```

Comment:
who added this?

not sure if we should do it

Reply:
I did. rocksdb docs: https://github.com/rust-rocksdb/rust-rocksdb?tab=readme-ov-file#compression-support says that if you use one compression algo, you don't need to enable all features. drops 11 crates from build.